CHANGELOG-0.5.md

```diff
@@ -1,3 +1,34 @@
+# Release notes for v0.5.1
+
+[Documentation](https://docs.k8s.io/docs/home)
+# Changelog since v0.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Add `process_start_time_seconds` metric. ([#106](https://github.com/kubernetes-csi/external-resizer/pull/106), [@Jiawei0227](https://github.com/Jiawei0227))
+- Adding configurable QPS and Burst to k8s client and use a separate client for leader election. ([#91](https://github.com/kubernetes-csi/external-resizer/pull/91), [@RaunakShah](https://github.com/RaunakShah))
```

Comment:
hmm something doesn't seem right with the release notes tool. these two changes were only in 1.0?

Reply:
The issue is related to https://github.com/kubernetes-csi/csi-release-tools/issues/7#issuecomment-705128427

Fixed this by manually updated the release notes.